src/components/TodoCounter.tsx

```diff
@@ -0,0 +1,11 @@
+import { memo } from 'react';
+
+type Props = {
+  value: number;
+};
+
+export const TodoCounter:React.FC<Props> = memo(({ value }) => (
+  <span className="todo-count" data-cy="TodosCounter">
+    {`${value} items left`}
```

Comment:
If there is one item, there should not be a plural in the text.

Reply:
Test thinks differently, and will fail on singular `item` text.